applications/EDVR/utils/utility.py

```diff
@@ -0,0 +1,71 @@
+#  Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
remove this file

Reply:
This file is needed when parsing yaml configure files, so it can't be removed